po/de.po

```diff
@@ -2400,6 +2402,8 @@ msgstr "Neue Konstruktionstechniken verbessern Turm"
 msgid "Replaces existing sensor tower"
 msgstr "Ersetzt bisherigen Sensorturm"
 
+# Da die Bezeichnung "spotter" für den verbesserten Sensorturm und nicht für eine Person steht ziehe ich
+# den Vorschlag "vorgeschobener Beobachter" zurück und bin dafür, die Übersetzung stehen zu lassen -alfred007
```

Comment:
Anmerkung: Es gab iirc in 1.12 (= inoffiziell) oder so mal ein VTOL mit Sensorturm, der ebenfalls "Spotter" hieß.